gammapy/maps/hpx.py

```diff
@@ -18,6 +18,7 @@
 # HPX_FITS_CONVENTIONS, HpxConv
 __all__ = [
     'HpxGeom',
+    'HpxToWcsMapping'
```

Comment:
Why did you make HpxToWcsMapping public?
Without any example in the docs, it's probably not very useful for users.

If you want it public, fine, then we'll try to improve the docs.
Concretely, apart from a missing working example in the class docstring, filling the input and return type of the arguments for `make_wcs_data_from_hpx_data` would be good:

![screen shot 2018-02-27 at 18 48 07](https://user-images.githubusercontent.com/852409/36745273-41172d26-1bef-11e8-80b1-d74bcdafb236.png)

Reply:
Ok I switched this back to private for now.  My thinking was that since we now have public methods that can accept `HpxToWcsMapping` arguments it would make sense to have this object be public as well.  However I agree that before doing this we should try to improve the docs for this class.